### DISCUSSION TITLE
SEBE- visualisation issu

### DISCUSSION THREAD

**Discussion by dounia43** [UMEP-dev/UMEP#388] [General]
hello,
I have the following message error every time I try to visualize the irradiance results using SEBE visualizer. 
when I click on (Visualise) I don't see results.
 
WARNING    Traceback (most recent call last):
              File "C:/Users/pc/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\UMEP\SEBEVisual\sun.py", line 521, in visualize
              dataset = gdal.Open(self.plugin_dir + '/data/temp.tif')
              File "C:\PROGRA~1\QGIS3~1.16\apps\Python37\lib\site-packages\osgeo\gdal.py", line 4048, in Open
              return _gdal.Open(*args)
             RuntimeError: C:/Users/pc/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\UMEP\SEBEVisual/data/temp.tif: No such file or directory 

![irradiance](https://user-images.githubusercontent.com/64792884/159885850-1611b40a-c540-47bc-b871-29cee3ce1679.png)

Many thanks,

**Comment by biglimp** [comment]
Can you please state your version of UMEP, QGIS and also click on "Stack trace" to see if you have any additional error messages.

Where you able to execute SEBE without any errors?

**Comment by dounia43** [comment]
UMEP version: 3.19.3
Python version  : 3.7.0
 QGIS version : 3.16.14-Hannover Hannover, 

**stack trace** 
RuntimeError: C:/Users/pc/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\UMEP\SEBEVisual/data/temp.tif: No such file or directory 
Traceback (most recent call last):
  File "C:/Users/pc/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\UMEP\SEBEVisual\sun.py", line 521, in visualize
    dataset = gdal.Open(self.plugin_dir + '/data/temp.tif')
  File "C:\PROGRA~1\QGIS3~1.16\apps\Python37\lib\site-packages\osgeo\gdal.py", line 4048, in Open
    return _gdal.Open(*args)
RuntimeError: C:/Users/pc/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\UMEP\SEBEVisual/data/temp.tif: No such file or directory

**Comment by biglimp** [comment]
Can you please submit your output data from your SEBE run (the whole folder as a zip-file).

**Comment by dounia43** [comment]
[SEBE.zip](https://github.com/UMEP-dev/UMEP/files/8341280/SEBE.zip)

**Comment by biglimp** [comment]
I cannot replicate your error. I can run your data without errors. I think you need to update your QGIS. The latest version of Python is 3.9.x and that was introduced as from QGIS 3.20 if I remember correctly.

With this said, your SEBE run was unsuccessful as you have minus values. I suspect that you only have global radiation in your input meteorological file and that you have not clicked in "estimate diffuse and direct shortwave components from global radiation". Do that and re-run your model.

I will move this issue over to Discussions instead.

**Reply by dounia43** [reply]
I have updated Qgis and it is working. thank you, sir
